src/ElectricPokemon.java

```diff
@@ -0,0 +1,45 @@
+public class ElectricPokemon extends Pokemon{
+    private String damageType;
+    private String charge;
+
+    public ElectricPokemon(String name, int number, int hp, String damageType, String charge) {
+        super(name, number, hp);
+        super.setType("Electric");
```

Comment:
mooi gebruik gemaakt van je constructor door hier standaard het type in vast te zetten